### ISSUE TITLE
NEXT: Accordion missing a11y

### ISSUE THREAD

**Issue by Hugos68** [skeletonlabs/skeleton#2760]
### Describe the feature in detail (code, mocks, or screenshots encouraged)

In #2370, I implemented the Accordion, but I forgot some crucial accordion specific role's, states and keyboard navigation.

This should still be done, there are some utils in #2754 that will come handy so let's wait for that to be merged first.

If you could assign this to me that would be appreciated.

### What type of pull request would this be?

Enhancement

### Provide relevant links or additional information.

Accordion spec: https://www.w3.org/WAI/ARIA/apg/patterns/accordion/

**Comment by Hugos68** [comment]
Closed by #2778
